src/main/java/dal/fatih/todoly/Main.java

```diff
@@ -1,18 +1,22 @@
 package dal.fatih.todoly;
 
 import java.text.DateFormat;
+import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.*;
 
 
-
 public class Main {
 
-   /* public static String dateFormatter(Date date) {
-        DateFormat dueDateFormat = new SimpleDateFormat("dd.MM.yyyy");
-        String resultDate = dueDateFormat.format(date);
-        return resultDate;
-    }*/
+    /* public static String dateFormatter(Date date) {
+         DateFormat dueDateFormat = new SimpleDateFormat("dd.MM.yyyy");
+         String resultDate = dueDateFormat.format(date);
+         return resultDate;
+     }*/
+    public static Date dateParser(String date) throws ParseException {
+        DateFormat dueDateParser = new SimpleDateFormat("dd/MM/yyyy");

```

Comment:
Create this as a static field in the class, otherwise it will be created every time you call this method.